client/App.jsx

```diff
@@ -1,15 +1,127 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import axios from 'axios';
+import CurrentProduct from './CurrentProduct.jsx';
+import FeedList from './FeedList.jsx';
 
+import styled from 'styled-components';
+
+
+///////////styled components///////////////
+// const styledItems = styled.styledItems`
+//   background-color: papayawhip;
+// `;
+
+// grid-template-rows: 50% 1fr 1fr 1fr  1fr;
+
+const Header = styled.div`
+  display: grid;
+  grid-template:
+    'img'
+    'img'
+    'img'
+    'img'
+    'img'
+    'img'
+    'img'
+    'img'
+    'img'
+    'img'
+    'img'
+    'img'
+    'img'
+    'img'
+    'img'
+    'img'
+    'Name'
+    'rating'
+    'price'
+    'condition';
+  text-align: center;
+  border: 0.5px solid #f3eeee;
+  font-family: montserratregular,Arial,Helvetica,sans-serif;
+  font-size: 10px;
+  `;
+
+const Name = styled.div`
+  grid-area: Name;
+  border: 0.5px solid #f3eeee;
+  background-color: lightgray;
+`;
+
+const Img = styled.div`
+  grid-area: img;
+  border: 0.5px solid #f3eeee;
+`;
+const Rating = styled.div`
+  grid-area: rating;
+  border: 0.5px solid #f3eeee;
+`;
+const Price = styled.div`
+  grid-area: price;
+  border: 0.5px solid #f3eeee;
+  background-color: lightgray;
+`;
+const Condition = styled.div`
+  grid-area: condition;
+  border: 0.5px solid #f3eeee;
+`;
+
+
+///////////////////////////////////////////
```

Comment:
Not at all necessary, but you may find it helpful to break your styled components out into their own file so you can keep them organized and reuse them across components. If you export each of the styled components individually, you can then just import the ones you need in a given component.

Reply:
Thank you for the tip!! I didn't think of that .. ^^;

I initially had them in the individual components file because they were all slightly different and therefore thought they were un-reusable. I discovered yesterday that with styled components, you can have a base style and add on different styles and overwrite them! 
Even without this feature, I agree that it will be much tidier if I had the styles in a separate file.  

(ALTHOUGH for some reason the 'grid-template' would not work for my different components, they weren't lining up and were jagged.. I did not figure out why that was, and ended up using grid-rows and assigning each item to a number :( )

Reply:
Now that I think of it, if I have this much code to style my components and need to have a separate file for it, there isn't any difference from just having a style.css to handle everything..

Reply:
I think your second comment is right on, and something I waver on a lot myself with styled components. I've started thinking of styling as a continuum with 'Giant CSS File of Doom' on one end, and styled components in every JSX file and/or inline styling on the other end. I think part of getting better as a front end developer is having an always improving sense of where on that continuum to work given the demands of the project in front of you. Definitely something I still have a long way to go on myself!